modules/vmware-csi-driver-reqs.adoc

```diff
@@ -21,3 +21,79 @@ If a third-party vSphere CSI driver is present in the cluster, {product-title} d
 ====
 The VMware vSphere CSI Driver Operator is supported only on clusters deployed with `platform: vsphere` in the installation manifest.
 ====
+
+You can create a custom role for the Container Storage Interface (CSI) driver, the vSphere CSI Driver Operator, and the vSphere Problem Detector Operator. The custom role can include privilege sets that assign a minimum set of permissions to each vSphere object. This means that the CSI driver, the vSphere CSI Driver Operator, and the vSphere Problem Detector Operator can establish a basic interaction with these objects. The following table details these minimum set of permissions.
+
+[IMPORTANT]
+====
+Installing an {product-title} cluster in a vCenter is tested against a full list of privileges as described in the "Required vCenter account privileges" section. By adhering to the full list of privileges, you can reduce the possibility of unexpected and unsupported behaviors that might occur when creating a custom role with a set of restricted privileges. 
+====
+
+.Minimum permissions for the storage components
```

Comment:
I think this should be added at the same level with "Minimum permissions for the Machine API", under the section "Minimum required vCenter account privileges".

Reply:
Thanks, @duanwei33 . Updated. I moved the storage table among the other tables.